exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/diagnosiskey/Validation.java

```diff
@@ -80,6 +81,14 @@ public static int validateRollingPeriod(int rollingPeriod) {
         return codes.stream().filter(code -> !code.equals("FI")).filter(Validation::validateISOCountryCode).collect(Collectors.toSet());
     }
 
+    public static Map<String, Boolean> validateISOCountryCodesWithoutFI(Map<String, Boolean> codes) {
+        return validateISOCountryCodesWithoutFI(codes.entrySet().stream().filter(Map.Entry::getValue)
+                .map(Map.Entry::getKey)
+                .collect(Collectors.toSet()))
```

Comment:
One level of processing ends here with the ending ) but the indentation remains the same. This could be made clearer with some well-placed linebreaks and indentation.

Reply:
For indents I'm using ideas defaults, it might be a good idea to make some project specific settings in some point, but not know. I'll let what I can do for this.